management/README.md

```diff
@@ -4,14 +4,90 @@ This directory contains the configuration needed to setup a management GKE clust
 
 Please refer to the latest [docs](https://master.kubeflow.org/docs/gke/deploy/management-setup/)
 
+## Install Instructions
+
+1. Fetch the management package:
+    ```bash
+    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@master ./
+    cd ./management
+    # Get upstream package
+    make get-pkg
+    ```
+1. Set values for the name, project, and location of your management cluster:
+    ```bash
+    kpt cfg set ./instance name NAME
+    kpt cfg set ./instance location LOCATION
+    kpt cfg set ./instance gcloud.core.project PROJECT
+    ```
+    * NAME is the cluster name of your management cluster. Management cluster
+    should be a different cluster from your Kubeflow cluster.
+
+        Note, NAME should
+        * start with a lowercase character
+        * only contain lowercase characters, numbers and `-`
+        * end with a number or character
+        * contain no more than 18 characters
+    * LOCATION is the management cluster's location, you can choose between regional or zonal.
+    * PROJECT is the Google Cloud project where you will create this management cluster.
+
+    It stores values you set in `./instance/Kptfile`. Commit the
+    changes to source control to preserve your configuration.
+
+    You can learn more about `kpt cfg set` in [kpt documentation](https://googlecontainertools.github.io/kpt/reference/cfg/set/).
+
+1. Create/apply the management cluster:
+    ```bash
+    make apply
+    ```
+1. Create a kubeconfig context for the cluster:
+    ```bash
+    make create-ctxt
+    ```
+1. Install the config connector:
+    ```bash
+    make apply-kcc
+    ```
+
 ## Upgrade Instructions
 
-### Upgrading management cluster from Kubeflow 1.1 to 1.2
+### Upgrading management cluster from Kubeflow v1.1
 
-1. Update your `management/Makefile` with the new one https://github.com/kubeflow/gcp-blueprints/blob/master/management/Makefile.
-2. Uninstall the Config Connector in management cluster by
+1. Use your management cluster's kubectl context:
+    ```bash
+    # Look at all your contexts
+    kubectl config get-contexts
+    # Select your management cluster's context
+    kubectl config use-context ${MGMTCTXT}
+    ```
+1. Check your existing config connector version:
+    ```bash
+    # For Kubeflow 1.1, it should be 1.15.0
+    $ kubectl get namespace cnrm-system -ojsonpath='{.metadata.annotations.cnrm\.cloud\.google\.com\/version}'
```

Comment:
Was `cnrm-system` set somewhere?
Trying this command in my setup, I don't have this namespace.

Reply:
It's in the management cluster

Reply:
I see. Thanks!